source/rest-v2.md

```diff
@@ -667,6 +684,268 @@ The JSON response returns a list of collections with the following attributes:
+curl -H "Authorization: ApiKey <api_key>" --data "{
+  "collection": {
+    "title": "New Collection"
+  }
+}"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+  {
+    "collection": {
+      "id": 42,
+      "title": "New Collection"
+    }
+  }
+```
+
+### Request body parameters
+
+The following fields need to be sent within a collection scope (see right for an example.)
+
+| Name                 | Type      | Required | Description
+| -------------------- | --------- | -------- | -----------------
+| title                | String    | Yes      | Collection Title
+
+# Pages
+
+## Add pages to publication
+
+Add pages from a given PDF file into a the specific publication
+
+### HTTP Request
+
+`POST https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages`
+
+### URL Parameters
+
+| Parameter        | Description
+| ---------        | --------------------------
+| Group ID         | The ID of a specific group
+| Publication ID   | The ID of a specific publication
+
+
+```shell
+curl --location 'https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages' \
+--header 'Authorization: ApiKey <api_key>' \
+--header 'Content-Type: application/json' \
+--data '{
+    "source_url": "https://some/file.pdf"
+  }'
+```
+
+### Request body parameters
+
+| Name                 | Type     | Required | Description
+| -------------------- | -------- | -------- | -----------
+| source_url           | String   | Yes      | URL where the PDF file resides. HTTP and HTTPS are accepted, and it needs to be a public accessible file
+| source_pages_numbers | Array    | No       | List of page numbers from pdf file to be added into the publication
+| position             | Integer  | No       | Page number where new pages should be added (by default add pages at the end)
+| extraction_options   | Object   | No       | Define settings to extract hotspots from the PDF file. See [extraction options](#extraction-options) for allowed fields
+
+### Response codes
+
+This endpoint returns the `200` response code.
+
+## Replace a publication page
+
+Replace a page on specific publication
+
+### HTTP Request
+
+`PUT https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages/<Page Number>`
+
+### URL Parameters
+
+| Parameter        | Description
+| ---------        | --------------------------
+| Group ID         | The ID of a specific group
+| Publication ID   | The ID of a specific publication
+| Page Number      | The page number of page to be replaced
+
+
+```shell
+curl --location --request PUT 'https://api.publitas.com/v2/groups/<Group ID>/publications/<Publication ID>/pages/<Page Number>' \
+--header 'Authorization: ApiKey <api_key>' \
+--header 'Content-Type: application/json' \
+--data '{
+  "page": {
+    "source_url": "https://some/file.pdf"
+  }
+}'
+```
+
+### Request body parameters
+
+The following fields need to be sent within a page scope (see right for an example.)
```

Comment:
`The following fields need to be sent within a page scope (see the right panel for an example):`